deploy.js

```diff
@@ -0,0 +1,139 @@
+const { SkynetClient, genKeyPairFromSeed } = require("@skynetlabs/skynet-nodejs");
+
+// ===============================
+// ===== CUSTOMIZABLE FIELDS =====
+// ===============================
+
+// URL of Skynet Portal you wish to use
+const PORTAL = process.env.SKYNET_PORTAL || "https://siasky.net";
+
+// Build directory.
+const BUILD_DIR = process.env.BUILD_DIR || "./dist";
+
+// Resolver datakey, e.g. "skynet-mysky". Allowed to be empty.
+const RESOLVER_DATA_KEY = process.env.RESOLVER_DATA_KEY || "";
+
+// Seed for generating and updating resolver skylink. Not allowed to be empty.
+const RESOLVER_SEED = process.env.RESOLVER_SEED || "";
+
+// A space-separated list of files that the portal should try when resolving a directory.
+const TRY_FILES = process.env.TRY_FILES || "index.html";
```

Comment:
you need to urlencode the file names yourself, I think that's fair trade off for such simplification

Reply:
I am not sure this approach is more self-explanatory than a JSONArray. I'm also not sure it will be self-evident for the devs that they need to URL-encode their filenames. In your example `/index.html` is not URL-encoded, for example. I am sure most devs won't think of encoding that.

If, on the other hand, they configure their `tryfiles` as a JSONArray we can do the encoding for them in the DAC.